src/module/module-collection.js

```diff
@@ -53,6 +53,14 @@ export default class ModuleCollection {
 
     parent.removeChild(key)
   }
+
+  isRegistered (path) {
+    const parent = this.get(path.slice(0, -1))
```

Comment:
I realized this line can throw an error if the parent module does not exist.
To handle this, we also need to add existance check in `get` method.